test/test_elements.py

```diff
@@ -108,27 +108,27 @@ class TestFunctionValues():
                                lambda x: (- 1 + x[0] + 3 * x[1], - 2 * x[1]),
                                lambda x: (-x[0], -2 + 3 * x[0] + 2 * x[1]),
                                lambda x: (2 * x[0], 1 - 3 * x[0] - x[1])]
-    reference_triangle_rt1 = [lambda x: (x[0], x[1]), lambda x: (1 - x[0], -x[1]),
-                              lambda x: (x[0], x[1] - 1)]
-    reference_triangle_rt2 = [lambda x: (-x[0] + 3 * x[0]**2, -x[1] + 3 * x[0] * x[1]),
-                              lambda x: (-x[0] + 3 * x[0] * x[1], -x[1] + 3 * x[1]**2),
-                              lambda x: (2 - 5 * x[0] - 3 * x[1] + 3 * x[0] * x[1] + 3 * x[0]**2,
-                                         -2 * x[1] + 3 * x[0] * x[1] + 3 * x[1]**2),
-                              lambda x: (-1.0 + x[0] + 3 * x[1] - 3 * x[0] * x[1], x[1] - 3 * x[1]**2),
-                              lambda x: (2 * x[0] - 3 * x[0] * x[1] - 3 * x[0] ** 2,
-                                         -2 + 3 * x[0] + 5 * x[1] - 3 * x[0] * x[1] - 3 * x[1]**2),
-                              lambda x: (-x[0] + 3 * x[0]**2,
-                                         1 - 3 * x[0] - x[1] + 3 * x[0] * x[1]),
-                              lambda x: (6 * x[0] - 3 * x[0] * x[1] - 6 * x[0]**2,
-                                         3 * x[1] - 6 * x[0] * x[1] - 3 * x[1]**2),
-                              lambda x: (3 * x[0] - 6 * x[0] * x[1] - 3 * x[0]**2,
-                                         6 * x[1] - 3 * x[0] * x[1] - 6 * x[1]**2)]
+    reference_triangle_rt1 = [lambda x: (-x[0], -x[1]), lambda x: (x[0] - 1, x[1]),
```

Comment:
Does these elements have a different dofs/basis now?

Reply:
Yes, we use the integral moment version of them now